.pre-commit-config.yaml

```
@@ -44,5 +44,5 @@ repos:
       additional_dependencies:
       - "types-PyYAML"
       - "types-requests"
-      - "pyfirecrest~=1.4.0"
-      - "aiida-core~=2.4.0"
+      - "pyfirecrest~=2.5.0"
+      - "aiida-core~=2.5.1.post0"
```

```suggestion
      - "aiida-core>=2.5.1"
```
Is the `post0` really needed, since we don't really have one? I think you want the `2.5.1` release but the `~=` operator allows also `2.5.0`.

---

In fact, this has to change to even higher numbers, so far this depends when [PR#6043](https://github.com/aiidateam/aiida-core/pull/6043) is going to be merged, then we have to put that as the minimum version of aiida.

---

I think it's needed to pre-commit, just to check if there is incompatibility between the packages.

>  args: [--config-file=pyproject.toml]

Thanks a lot! I wasn't aware of this. It's fixed now.